app/controllers/agent/HomeController.scala

```diff
@@ -54,7 +54,7 @@ class HomeController @Inject()(home: Home,
       nextPaymentOrOverdue = nextPaymentOrOverdue,
       nextUpdateOrOverdue = nextUpdateOrOverdue,
       overduePaymentExists = overduePaymentExists,
-      paymentEnabled = isEnabled(Payment),
```

Comment:
Please remove this from the view instead of setting it to true

Reply:
done